docs/chapters/getting-started/try-it-out.rst

```diff
@@ -0,0 +1,40 @@
+Try it out
+==========
+
+To easily try PELUX you can download a ready-made boot image. Either a release
+version or the latest nightly build.
+
+Make sure you have one of the `supported hardware setups <hardware-setup.html#supported-hardware>`_
+and simply boot with the downloaded image. 
+
+Raspberry Pi
+------------
+1. Get the image you want
+
+  * `Latest release <https://pelux.io/releases/#images>`_
+  * `Latest nightly <https://pelux.io/jenkins/view/NIGHTLY/>`_
+
+2. Burn the image to a 16 GB (or larger) SD-card
```

Comment:
Consider link to section presented in swf-blueprint/docs/articles/baseplatform/deploying-image.rst which explains how the image is deployed to the SD card/USB stick.